### DISCUSSION TITLE
Welcome to mseuniverse Discussions!

### DISCUSSION THREAD

**Discussion by fredvs** [mse-org/mseuniverse#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

 

![msegui_64](https://github.com/mse-org/mseuniverse/assets/3421249/3541a25a-86dc-4ff5-9b72-0155025b1bcd)

![msegui](https://github.com/mse-org/mseuniverse/assets/3421249/cacb636b-912d-4bf7-ab6a-4b16c8ad5513)

**Comment by lainz** [comment]
Hi, I made a button and a progress bar some time ago. Hope it inspires others to use bgrabitmap with mse.

**Comment by fredvs** [comment]
Hello Lainz.

Yes, of course, all your BgraBitmap-widgets are highly welcome.
(I cannot find where you have stocked that project).

**Reply by lainz** [reply]
Thanks. Here it is: https://github.com/bgrabitmap/bgragui

**Reply by fredvs** [reply]
Ha, perfect.
Thanks, I will re-jump into it.
